scripts/afterRelease.sh

```diff
@@ -0,0 +1,24 @@
+brew install hub
+yarn typedoc --json docs.json
+node scripts/postProcessDocs.js
+
+packageVersion=$(node -e "const fs = require('fs');\
+const packageFile = fs.readFileSync('./package.json');\
+const packageJson = JSON.parse(packageFile.toString());\
+console.log(packageJson.version)")
+branchName="feat/updateSDKJsExamples$packageVersion"
```

Comment:
We can't get conflict here? So we only update when we release a new version?

Reply:
correct, we only update after we release. means, we won't have the same branch name